workers/unity/Assets/Tests/PlaymodeTests/Performance/ComponentUpdateTests.cs

```diff
@@ -0,0 +1,58 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.TestUtils;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+
+namespace Improbable.Gdk.PlaymodeTests
+{
+    [TestFixture]
+    public class ComponentUpdateTests : MockBase
+    {
+        private const string WorkerType = "TestWorkerType";
+        private const long EntityId = 100;
+
+        protected override MockWorld.Options GetOptions()
+        {
+            return new MockWorld.Options
+            {
+                WorkerType = WorkerType
+            };
+        }
```

Comment:
I don't _think_ you need this block

Reply:
I don't, it's leftover from copying a different test